main.go

```diff
@@ -0,0 +1,5 @@
+package main
```

Comment:
file is not needed as we already have a working main.go

Reply:
ok i delete this